### DISCUSSION TITLE
Execution order of the queries

### DISCUSSION THREAD

**Discussion by Etkiny** [Budibase/budibase#5033] [Help]
Can I execute my Queries in an order, so that if the first query shows an error, It should not pass to the second and stop there showing an error?

Currently, my queries works in order, however they execute without examining If th previous one works or not.

If there is an error in one of the queries, It should not pass to the next one. 

Thank you so much! 

<img width="664" alt="image (9)" src="https://user-images.githubusercontent.com/83818438/159655318-69321161-d17d-45ee-980c-d9e9ded9a473.png">

**Comment by melohagan** [comment]
Hi @Etkiny 

Currently this is not possible, but there is an in progress issue out for a **Continue If** step for actions: https://github.com/Budibase/budibase/issues/4143

When this is released, you will be able to use it in conjunction with `{{ Action X.Query Result }}` and read if the response body is empty or contains an error.

More on action referencing here: https://docs.budibase.com/docs/referencing
